src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -50,7 +50,7 @@ const Covid19Container = () => {
     const overviewRequest = useRef(null);
     const allSubmissionDatesRequest = useRef(null);
     const dispatch = useDispatch();
-    const defCodes = useSelector((state) => state.covid19.defCodes, isEqual);
+    const defCodes = useSelector((state) => state.covid19.defCodes.filter((c) => c.disaster === 'covid_19'), isEqual);
```

Comment:
This should not be the case. Defcodes in the COVID 19 filter in Redux should be COVID Defcodes.